### ISSUE TITLE
Wordpress rss image

### ISSUE THREAD

**Issue by theScootz01** [pavelk2/social-feed#229]
I created a blog and im pulling it via the rss, but cant seem to figure out how to pull the feature image and not the images in post itself.

**Comment by HartLarsson** [comment]
please search in previous issues, i've posted a Solution to another user that have same problem

https://github.com/pavelk2/social-feed/issues/225

**Comment by theScootz01** [comment]
Thank you, but it pulls images from the content and not the feature image at the top set by wordpress.
Or am i missing something?

**Comment by HartLarsson** [comment]
Yes, because i've not personally implemented that kind of "extraction"
What is doing is take esplicited rss feed image or if not found use the first one from the body of the article.
